### PR TITLE
first html page

### PR DESCRIPTION
**Pull request by blam-codes**
Git-16 Added index.html page

### CODE REVIEW

File: index.html  [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<! doctype html>
+<head>
+	<title> Bubble Tea site</title>
```
Comment (dlamdevelops) [inline index.html :3]: Please remove space after <title> and all similar extra spaces on other lines.
Will refer to these issues as "space tag issues"

Comment (dlamdevelops) [inline index.html :3]: Please use actual tabs throughout entire document.... these look like spaces.  Put this through an html prettifier online and then set up Visual Studio to do tab with two spaces.  Look this up online.

File: index.html  [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+</head>
+<body>
+	< div class=“topnav”>
```
Comment (dlamdevelops) [inline index.html :6]: space tag issue before < and div

File: index.html  [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+		<img src =“ ” ALT=“logo” Width=600 Height = 400”/>
```
Comment (dlamdevelops) [inline index.html :7]: ALT needs to be alt
width and height need to be in style="width:600px; height:400px" format

File: index.html  [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+		<a class= “active” href=#home”>Home</a>
```
Comment (dlamdevelops) [inline index.html :8]: space tag issue after = and "active"

File: index.html  [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+		<a href=#news”>News</a>
+		< a href=“about”>About</a>
```
Comment (dlamdevelops) [inline index.html :10]: space tag issue

File: index.html  [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+		<a href=“contact”>Contact</a>
+	</div>
+	<div>
+	<img src =“”  Alt=“description of image”/>
```
Comment (dlamdevelops) [inline index.html :14]: space tag issues
Alt needs to be alt
 is not self-closing.... so drop the extra "/" at the end

File: index.html  [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+</div>
+
+<div> 
+	<p> text </p>
+</div>
+
+
+<div>
+	<p>text </p>
```
Comment (dlamdevelops) [inline index.html :23]: space tag issue

File: index.html  [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+</div>
+
+<div>
+	<img src =“”  Alt=“description of image”/>
```
Comment (dlamdevelops) [inline index.html :27]: space tag issue

File: index.html  [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+</div>
+
+<div>
+	<p> Text </p>
```
Comment (dlamdevelops) [inline index.html :31]: space tag issue

File: index.html  [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+	<button>More </button>
```
Comment (dlamdevelops) [inline index.html :32]: Get rid of space after More

File: index.html  [hunk 11]
```diff
@@ -0,0 +1,42 @@
...
+
+</div>
+
+<footer>
+	<p>copyright2020</p>
```
Comment (dlamdevelops) [inline index.html :37]: add space between copyright and 2020

File: index.html  [hunk 12]
```diff
@@ -0,0 +1,42 @@
...
+	<p>Terms of condition</p>
```
Comment (dlamdevelops) [inline index.html :38]: should this and the next Contact us be links?

File: index.html  [hunk 13]
```diff
@@ -0,0 +1,42 @@
...
+	<p>Contact us</p>
+</footer>
+	</body>
```
Comment (dlamdevelops) [inline index.html :41]: again, tab issues with body here...need to put this entire file through an html prettifier

**Review by dlamdevelops — CHANGES_REQUESTED** [review]
Lots of cleaning up issues around spacing
Nothing wrong with tags, per se, except in footer

File: index.html  (hunk 1 above)
Comment (blam-codes) [reply index.html :3]: Hi David, I have did changes to spacing. Can you confirm that you where able to pull the changes? Thank you.

File: index.html  [hunk 14]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<head>
+	<title>Bubble Tea site</title>
+</head>
+<body>
+		< div class="topnav">
```
Comment (dlamdevelops) [inline index.html :6]: Spacing issue with div and angle bracket

Comment (dlamdevelops) [inline index.html :6]: Fix all unnecessary indentation across entire file...
Lines 2 and 3 should show you the amount of correct space between child and parent tags

File: index.html  [hunk 15]
```diff
@@ -0,0 +1,43 @@
...
+			<img src="" alt="logo" style="width:600px; height:400px"/>
+			<a class="active" href=#home">Home</a>
+			<a href=#news">News</a>
+			<a href="about">About</a>
```
Comment (dlamdevelops) [inline index.html :10]: your hrefs need to be url's or anchor tags like line 9

File: index.html  [hunk 16]
```diff
@@ -0,0 +1,43 @@
...
+			<a href=“contact”>Contact</a>
```
Comment (dlamdevelops) [inline index.html :11]: same as comment above on anchor tags or live urls

File: index.html  [hunk 17]
```diff
@@ -0,0 +1,43 @@
...
+		</div>
+		
+    <div>
+      <img src=""alt="description of image">
```
Comment (dlamdevelops) [inline index.html :15]: spacing issue

File: index.html  [hunk 18]
```diff
@@ -0,0 +1,43 @@
...
+		</div>
+
+		<div> 
+		  <p>text</p>
+		</div>
+    
+    <div>
+		  <p>text</p>
+		</div>
```
Comment (dlamdevelops) [inline index.html :24]: your closing and end tags (in this case <div> and </div> need to align...

File: index.html  [hunk 19]
```diff
@@ -0,0 +1,43 @@
...
+			<img src="" alt="description of image"/>
+		</div>
+
+		<div>
+			<p>Text</p>
+			<button>More</button>
+    </div>
+
+		<footer>
+				<a href="url">copyright 2020</a>
```
Comment (dlamdevelops) [inline index.html :36]: "url" is not either an anchor tag or a live url 
please fix for all hrefs

**Review by dlamdevelops — CHANGES_REQUESTED** [review]
Indentations are still off across the entire index.html file
Lines 2 and 3 show the perfect amount of indentation for parent and child element tags.

**Comment by blam-codes** [comment]
It was a placement holder.

File: index.html  [hunk 20]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<head>
+	<title>Bubble Tea site</title>
+</head>
+<body>
+	<div class="topnav">
```
Comment (dlamdevelops) [inline index.html :6]: Please ensure that ending and closing tags for the divs are all vertically aligned.

CORRECT:
<div>
</div>

INCORRECT:
<div>
   </div>

All <div>'s and their closing tags should all be vertically aligned.

**Review by dlamdevelops — CHANGES_REQUESTED** [review]
Still having issues with alignment of div opening and closing tags.
Use the vertical lines in your editor to help you.
You should never have to use the spacebar... only tab for indents.

File: index.html  [hunk 21]
```diff
@@ -0,0 +1,43 @@
...
+		<img src="" alt="logo" style="width:600px; height:400px"/>
+		<a class="active" href=#home">Home</a>
+		<a href=#news">News</a>
+		<a href=#about">About</a>
+		<a href=#contact”>Contact</a>
+	</div>
+		
+    <div>
```
Comment (dlamdevelops) [inline index.html :14]: You can see that these are not aligned.  All closing and opening tags need to be aligned, per level of the html hierarchy.

File: index.html  [hunk 22]
```diff
@@ -0,0 +1,43 @@
...
+		<img src=""alt="description of image">
+	</div>
+
+	<div> 
+		<p>text</p>
+	</div>
+    
+    <div>
```
Comment (dlamdevelops) [inline index.html :22]: Same issue with indent as above

File: index.html  [hunk 23]
```diff
@@ -0,0 +1,43 @@
...
+	</div>
+
+	<div>
+		<img src="" alt="description of image"/>
+	</div>
+
+	<div>
+		<p>Text</p>
+		<button>More</button>
+    </div>
```
Comment (dlamdevelops) [inline index.html :33]: Same issue as above

File: index.html  [hunk 24]
```diff
@@ -0,0 +1,43 @@
...
+
+	<footer>
+		<a href=#copyright">copyright 2020</a>
+		<a href=#terms">Terms of Conditions</a>
+		<a href=#contact">Contact Us</a>
+	</footer>
+	
+ </body>
+
```
Comment (dlamdevelops) [inline index.html :42]: extra space

File: index.html  [hunk 25]
```diff
@@ -0,0 +1,43 @@
...
+		<p>Text</p>
+		<button>More</button>
+    </div>
+
+	<footer>
+		<a href=#copyright">copyright 2020</a>
+		<a href=#terms">Terms of Conditions</a>
+		<a href=#contact">Contact Us</a>
+	</footer>
+
```
Comment (dlamdevelops) [inline index.html :40]: extra space

File: index.html  [hunk 26]
```diff
@@ -0,0 +1,43 @@
...
+	<div class="topnav">
+		<img src="" alt="logo" style="width:600px; height:400px"/>
+		<a class="active" href=#home">Home</a>
+		<a href=#news">News</a>
+		<a href=#about">About</a>
+		<a href=#contact”>Contact</a>
+	</div>
+		
+    <div>
+		<img src=""alt="description of image">
```
Comment (dlamdevelops) [inline index.html :15]: always need 1 space between properties on an HTML element

**Review by dlamdevelops — CHANGES_REQUESTED** [review]
Requires additional spacing work
@blam-codes

**Review by dlamdevelops — APPROVED** [review]
Approved.  Please squash

**Pull request merged**